sigh/lib/sigh/download_all.rb

```diff
@@ -1,13 +1,15 @@
 module Sigh
   class DownloadAll
     # Download all valid provisioning profiles
-    def download_all
+    def download_all(options, args)
+      download_xcode_profiles = options.download_xcode_profiles
```

Comment:
Does this work? I thought you need to access this variable using `options[:download_xcode_profiles]`